alembic/versions/df47027ea829_update_events_and_room_models.py

```diff
@@ -0,0 +1,38 @@
+"""update events and room models
+
+Revision ID: df47027ea829
+Revises: 99ec6a5fa861
+Create Date: 2019-03-06 12:16:34.744305
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'df47027ea829'
+down_revision = '99ec6a5fa861'
```

Comment:
Hello @caveinn , We shall need to update this revision due to the current changes on v2

Reply:
This has been handled